config/default.json

```diff
@@ -18,18 +18,55 @@
     "CREATE_RETRY_INTERVAL_MILLIS": 200,
     "DEBUG": false
   },
+  "WINDOW_AGGREGATION": {
+    "RETRY_COUNT": 3,
+    "RETRY_INTERVAL": 3000
```

Comment:
Should this be `RETRY_INTERVAL_MILLIS` or `_SECONDS` as in other examples in this file?

Reply:
Hi Lewis. There's no standard on this. You can find other settings in this file which indicate milliseconds, without holding this suffix, added by Rajiv 4 months ago. E.g.:
`"consumeTimeout": 1000`
In central-ledger default.json you may also find:
```
  "SIDECAR": {
   ...
    "CONNECT_TIMEOUT": 45000,
    "RECONNECT_INTERVAL": 5000
  },
```
In ml-api-adapter:
```
  "ENDPOINT_CACHE_CONFIG": {
    "expiresIn": 180000,
    "generateTimeout": 30000
  },
```
I wouldn't be bothered about values ending with 000 as they clearly indicate milliseconds.

Reply:
Sure no problem. I just saw the other examples. No matter either way.

Reply:
@lewisdaly @ggrg @elnyry I think a story should be created to standardise the way env variables are written and apply it to all libraries

Reply:
why, so that we risk breaking everything ;-) ? 

On a serious note, sure we can have one, but my only concern is that it will add to the pile of backlog items and not get looked at any time soon. IF someone wants to volunteer for it, yes we can have the issue right now.

Reply:
Sure @elnyry maybe the community can pick it up